.github/workflows/rust-cubesql.yml

```diff
@@ -196,6 +196,13 @@ jobs:
           CUBEJS_NATIVE_INTERNAL_DEBUG: true
         working-directory: ./packages/cubejs-testing
         run: yarn smoke:cubesql
+      - name: Store build artifact for dev image
+        if: (matrix.python-version == '3.11') && (matrix.target == 'x86_64-unknown-linux-gnu')   # No need to create many versions
+        uses: actions/upload-artifact@v4
+        with:
+          name: "dev-cubejs-backend-native-x86_64"    # this name is referenced in master.yml->docker-image-dev
+          path: ./packages/cubejs-backend-native
```

Comment:
This will upload target directory. You need to upload `index.node` file

Reply:
fixed